packages/fluentui/accessibility/src/behaviors/Tree/treeBehavior.ts

```diff
@@ -19,6 +19,9 @@ export const treeBehavior: Accessibility<TreeBehaviorProps> = props => {
         role: 'tree',
         'aria-labelledby': props['aria-labelledby'],
         tabIndex: -1,
+        ...(props.selectable && {
+          'aria-multiselectable': true,
```

Comment:
Please could you update the specification (or add it via new a11y tests created by Milan)?

Reply:
Resolved by this [commit](https://github.com/microsoft/fluentui/pull/15147/commits/da17607de83c312772121a8637f11fbcaa9e3d98)